.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Authors of Nimbus
+
+name: Conformance tests
+
+permissions: {}
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "release*"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+jobs:
+    chainsaw-tests:
+        runs-on: ubuntu-latest
+        permissions:
+            packages: read
+        strategy:
+            fail-fast: false
+        steps:
+            - name: Checkout
+              uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+              # install tools
+            - name: Install helm
+              id: helm
+              uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
+              with:
+                token: ${{ secrets.GITHUB_TOKEN }}
+              
+              # install go
+
+            - name: Install go
+              uses: actions/setup-go@v5
+              with:
+                go-version: '1.21.7' # The Go version to download (if necessary) and use.
+              
+
+            # install chainsaw
+
+            - name: Install chainsaw
+              uses: kyverno/action-install-chainsaw@8307cf9038333ce168dd4339ae24f9ab16ab7a00 # v0.1.4
+
+            # create kind cluster
+
+            - name: Create k8s Kind Cluster
+              uses: helm/kind-action@v1
+
+            - name: Run Tests
```

Comment:
Instead of directly running the nimbus using a make target, execute tests against the locally built docker image. For e.g.,
https://github.com/kubearmor/KubeArmor/blob/1e31e1ae46aa99629410239afac16e98ada1fa7d/.github/workflows/ci-latest-release.yml#L76-L88

Reply:
Build a local image using `make docker-build` and then deploy it from the `deployments/nimbus` directory like:
```shell
helm upgrade --install nimbus-operator . -n nimbus --create-namespace --set image.pullPolicy=Never
```

Then execute tests against that local image.

Reply:
Since our CRDs are not stable yet, so we've to directly install and run nimbus using make targets:

- `make install` to install CRDs, and
- `make run` to run nimbus.